src/main/clojure/clara/rules/engine.cljc

```diff
@@ -6,9 +6,12 @@
             [clojure.string :as string]
             [clara.rules.memory :as mem]
             [clara.rules.listener :as l]
-            [clara.rules.platform :as platform]
+            #?(:clj [clara.rules.platform :as platform :refer [eager-for]]
```

Comment:
I'm always just not a fan of `:refer` and I don't think we really use it much in this ns.
This could be written as:
```clj
#?(:clj [clara.rules.platform :as platform]
   :cljs [clara.rules.platform :as platform :include-macros true])
```
I think you can even get away with
```clj
[clara.rules.platform :as platform #?@(:cljs [:include-macros true])]
```
but I don't tend to see that for whatever reason.

See http://blog.fikesfarm.com/posts/2016-03-01-clojurescript-macro-sugar.html for more on the `ns` "sugar"

But perhaps others like the `:refer` and if that is the case, so be it. I just think "alias everything" improves readability overall since it is immediately clear when a var is "foreign" to a ns or not.

And could then just use the alias as always.